packages/react-composites/src/composites/ChatComposite/FileDownloadCards.tsx

```diff
@@ -48,20 +52,34 @@ export const FileDownloadCards = (props: FileDownloadCards): JSX.Element => {
             fileName={truncatedFileName(file.name, truncateLength)}
             key={file.name}
             fileExtension={extension(file.name)}
-            actionIcon={<DownloadIconTrampoline />}
+            actionIcon={
+              showSpinner ? (
+                <Spinner size={SpinnerSize.medium} aria-live={'assertive'} />
+              ) : (
+                <ChatCompositeIcon iconName="Download" />
+              )
+            }
             actionHandler={() => {
-              props.downloadHandler && props.downloadHandler(userId, file);
+              if (props.downloadHandler) {
+                setShowSpinner(true);
+                props
+                  .downloadHandler(userId, file)
+                  .then((value: URL | FileDownloadErrorMessage) => {
+                    if (value instanceof URL) {
+                      window.open(value.toString(), '_blank', 'noopener,noreferrer');
```

Comment:
`.toString()` is unecessary here, window.open should accept a URL type

Reply:
done